api/v1alpha3/config/config.go

```diff
@@ -0,0 +1,171 @@
+package config
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/eraser-dev/eraser/api/v1alpha3"
+	"github.com/eraser-dev/eraser/version"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+var defaultScannerConfig = `
+cacheDir: /var/lib/trivy
+dbRepo: ghcr.io/aquasecurity/trivy-db
+deleteFailedImages: true
+deleteEOLImages: true
+vulnerabilities:
+  ignoreUnfixed: true
+  types:
+    - os
+    - library
+securityChecks: # need to be documented; determined by trivy, not us
+  - vuln
+severities:
+  - CRITICAL
+  - HIGH
+  - MEDIUM
+  - LOW
+`
+
+type Manager struct {
+	mtx sync.Mutex
+	cfg *v1alpha3.EraserConfig
+}
+
+func (m *Manager) Read() (v1alpha3.EraserConfig, error) {
```

Comment:
I believe you can remove these methods and the `NewManager` function. Only the `unversioned` package needs those. For example, the corresponding functions from the `v1alpha2/config` package are not actually called anywhere.

I think the functions aren't even needed for the conversion generation either. You can remove these, and the corresponding ones in `v1alpha1` and `v1alpha2` configs.